### PR TITLE
add llama2 examples for smoothquant

### PR DESCRIPTION
**Pull request by chensuyue**
## Type of Change

examples

## Description

llama2-7b and llama2-13b SmoothQuant with Intel® Neural Compressor and Intel® Extension for PyTorch.

## Expected Behavior & Potential Risk

Run llama2-7b and llama2-13b with INC SmoothQuant recipes and get the int8 model with expected accuracy.  

## How has this PR been tested?

Local test

## Dependency Change?

No

### CODE REVIEW

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/run_llama2_sq.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+if args.sq_recipes == "llama2-7b":
+    op_type_dict = {"add": {"weight": {"dtype": ["fp32"]}, "activation": {"dtype": ["fp32"]}}}
+    excluded_precisions = [] if args.int8_bf16_mixed else ["bf16"]
+    op_name_dict = {}
+    recipes = {"smooth_quant": True, "smooth_quant_args": {'alpha': 'auto', 'folding': False, 'default_alpha': 0.8,
+                                                           'auto_alpha_args': {"alpha_min": 0.8, "alpha_max": 0.99,
+                                                                               "alpha_step": 0.01,
+                                                                               "shared_criterion": "mean"}}}
+else:
+    print("TODO")
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/run_llama2_sq.py:209]: @yintong-lu please add the recipes for llama2-13b.

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+```
+## Llama-2-13b
+```bash
+python run_llama2_sq.py \
+    --model-id meta-llama/Llama-2-13b-hf \
+    --batch-size 56 \
+    --sq-recipes "llama2-13b"
+```
+> Notes:  
+> - INT8 model will be saved into "./saved_results", and the output model can be loaded and evaluated by IPEX.
```
Comment (thuang6) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/README.md:28]: How about listing generated files in saved_results. In some case,  user may only take qconfig file which reflects the tuning result.  for example, in IPEX LLM example (https://github.com/intel-innersource/frameworks.ai.pytorch.ipex-cpu/tree/release/2.1/examples/cpu/inference/python/llm#static-quantization-int8), it does quantization itself but take our tuning result config as input

**Review by thuang6 — COMMENTED** [review]
another issue is that in our [SQ doc](https://github.com/intel/neural-compressor/blob/master/docs/source/smooth_quant.md#validated-models) and LLM [llama2 example](https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm#llama2-7b13b30b)   we have different recipes and sample code

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/README.md (hunk 2 above)
Comment (chensuyue) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/README.md:29]: OK, I will list the files in saved_results

**Comment by chensuyue** [comment]
> another issue is that in our [SQ doc](https://github.com/intel/neural-compressor/blob/master/docs/source/smooth_quant.md#validated-models) and LLM [llama2 example](https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm#llama2-7b13b30b) we have different recipes and sample code

The difference of LLM [llama2 example](https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm#llama2-7b13b30b) and this new script is they use the different model load method, and the output of the first example can't use ipex to do the evaluate, we use ITREX to evaluate.

**Review by thuang6 — APPROVED** [review]

**Pull request merged**